cmake/sysbuild/sign_nrf54h20.cmake

```diff
@@ -4,6 +4,45 @@
 
 include(${ZEPHYR_NRF_MODULE_DIR}/cmake/sysbuild/bootloader_dts_utils.cmake)
 
+function(check_merged_slot_boundaries merged_partition images)
+  # Predefine the MCUboot header size.
+  set(MCUBOOT_HEADER_SIZE 0x800)
```

Comment:
the great thing here is this is in the **post** cmake function, so all images have been configured, you are free to load the Kconfig tree of the main application and get this value directly from the Kconfig value meaning it can be dynamically updated in images without being hardcoded :)

Reply:
Unfortunately, this value is no longer present there: https://github.com/nrfconnect/sdk-zephyr/pull/3310/commits/3278b2a5647a4d68ea287eadc593841f4bcdd455

The thing is that the `ROM_START_OFFSET` does not have prompt, thus it cannot be managed like the `ROM_END_OFFSET`. Also - using `ROM_START_OFFSET` would make the `slot0_partition` and `cpuapp_slot0_partition` start at the same location, causing errors while parsing DTS.